UM/FileHandler/ReadFileJob.py

```diff
@@ -24,10 +25,17 @@ def getFileName(self):
         return self._filename
 
     def run(self):
+        # HACK: Packages shouldn't be checked for a reader. Maybe someday we build a reader. Maybe not. But for now,
+        # this hack is the only solution.
+        filename, file_extension = os.path.splitext(self._filename)
+        if file_extension == ".curapackage":
```

Comment:
This used to be a hack in `Cura.qml`, check this commit for example:
https://github.com/Ultimaker/Cura/commit/0f966115e65762149d61009b4968598a5bb33c20#diff-4b601941ef414d7ed8e6b673a2d46752

Reply:
OK, that way doesn't work in this case..